contrib/app/SpotAnalysis/PeakFlux.py

```diff
@@ -89,6 +100,20 @@ def run(self):
             # condensed csv file.
             parser = saoap.SpotAnalysisOperableAttributeParser(result, self.spot_analysis)
 
+    def get_bcs_origin(self, operable: SpotAnalysisOperable):
```

Comment:
We should discuss moving this file other files being actively developed out of contrib. These should likely have have a doc-string and unit-test.

Reply:
I agree that it should have docstrings and maybe also unit tests, but I believe it still falls under the idea of "contrib" code:

"This is experimental code, still in early stages of exploration and development."

Once it gets more applicable to general situations, then I agree it should be moved out of contrib. What do you think? Is this reasoning good or am I missing something?